routes/web.php

```diff
@@ -17,12 +17,8 @@
 
 Auth::routes();
 
-Route::get('/home', 'HomeController@index')->name('home');
-Auth::routes();
-
-Route::get('/home', 'HomeController@index')->name('home');
-
-// TODO: move to controller
-Route::get('/print', function () {
-    return view('print.app');
-});
\ No newline at end of file
+Route::middleware(['auth'])->group(function () {
+    Route::get('/home', 'HomeController@index')->name('home');
+    Route::get('/print', 'PrintController@index')->name('print');
+    Route::post('/print/modify_balance', 'PrintController@modify_balance');
```

Comment:
No authorization needed?

Reply:
They are grouped under the 'auth' middleware which enforces authorization.

Reply:
OK. I was thinking about the checking of admin permissions for modifying print balance. But as I see, permission management is not part of this issue and will only be added later.